endpointmanager/pkg/endpointmanager/fhirendpoint_test.go

```diff
@@ -141,17 +156,17 @@ func Test_FHIREndpointEqual(t *testing.T) {
 	}
 	endpoint2.URL = endpoint1.URL
 
-	endpoint2.OrganizationNames = []string{"Other 1"}
+	endpoint2.OrganizationList[0].OrganizationName = "Other 1"
 	if endpoint1.Equal(endpoint2) {
 		t.Error("Did not expect endpoint1 to equal endpoint 2. OrganizationNames should be different.")
 	}
-	endpoint2.OrganizationNames = endpoint1.OrganizationNames
+	endpoint2.OrganizationList[0].OrganizationName = endpoint1.OrganizationList[0].OrganizationName
 
-	endpoint2.NPIIDs = []string{"Other 1"}
+	endpoint2.OrganizationList[0].OrganizationNPIID = "Other 1"
 	if endpoint1.Equal(endpoint2) {
 		t.Error("Did not expect endpoint1 to equal endpoint 2. NPIIDs should be different.")
 	}
-	endpoint2.NPIIDs = endpoint1.NPIIDs
+	endpoint2.OrganizationList[0].OrganizationNPIID = endpoint1.OrganizationList[0].OrganizationNPIID
 
```

Comment:
I think it would be good to add some other edge case tests of the `OrganizationListEquals` function so that we verify that if the organizations are the same but in a different order, it doesn't register that as a change, and if the lists have the same organizations, but one has a few extra organizations, that is also a change.

Reply:
Added two new tests, one that checks to make sure endpoint1 still equals endpoint2 even if their organization lists are not in the same order, and one that checks to make sure endpoint1 does not equal endpoint2 if one of them has more organizations in their organization list than the other